apps/client/src/components/Modals/NewChapterModal.svelte

```diff
@@ -26,15 +26,20 @@
           },
         },
       });
-      console.log('New lesson added successfully!');
-      document.getElementById('close')?.click();
+      console.log('New chapter added successfully!');
```

Comment:
You could use a toast

Reply:
This is outside the scope of the button refactor, I think it deserves its own pr to handle different states across the entir app.

Reply:
Valid!